### PR TITLE
NLog SentryTarget - Support for BreadCrumb Category. Default Logger Name

### PR DESCRIPTION
**Pull request by snakefoot**

### CODE REVIEW

File: src/Sentry.NLog/SentryTarget.cs [hunk 1]
```diff
@@ -377,6 +387,9 @@ protected override void Write(LogEventInfo logEvent)
             if (logEvent.Level >= Options.MinimumBreadcrumbLevel)
             {
                 var breadcrumbFormatted = RenderLogEvent(BreadcrumbLayout, logEvent);
+                var breadcrumbCategory = RenderLogEvent(BreadcrumbCategory, logEvent);
+                if (string.IsNullOrEmpty(breadcrumbCategory))
+                    breadcrumbCategory = null;
```
Comment (bruno-garcia) [inline src/Sentry.NLog/SentryTarget.cs:393]:
```suggestion
                if (string.IsNullOrEmpty(breadcrumbCategory))
                {
                    breadcrumbCategory = null;
                }

```

**Comment by bruno-garcia** [comment]
@snakefoot Looks like there's a broken test.

**Comment by snakefoot** [comment]
> @bruno-garcia  Looks like there's a broken test.

Yes there was a test that expected BreadCrumb-Category to be null.

**Comment by bruno-garcia** [comment]
Thanks @snakefoot !

**Comment by snakefoot** [comment]
Happy to help. Have a great weekend.

**Pull request merged**